src/Delegates/HandlesSystem.php

```diff
@@ -31,4 +31,9 @@ public function stats(): array
     {
         return $this->stats->show();
     }
+
+    public function generateTenantToken($searchRules, ?array $options = []): string
```

Comment:
In the other SDKs I created a new "module"/"reference" for it, because for me also it didn't find some good place to put it!
I am pretty sure that will change in the future if the users complain about the validations.
We could live it here and in the Endpoints as well, since I don't know either a better place to put it!